test/REQUIRE

```diff
@@ -1 +1 @@
-ValidatedNumerics 0.5.0
+TaylorSeries 0.4.0
```

Comment:
Though, since `using TaylorIntegration` uses `TaylorSeries` (but is not exported) this may not be needed. The only difference is that it may be required to use things as `TaylorSeries.Taylor1`, as it was before. I don't know what's better.